miniconda3-nanoserver/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/windows/servercore:1809 AS servercorebase
+
+WORKDIR /tmp
```

Comment:
I don't think this is needed

Reply:
Doesn't harm and show explicitly what the workdir is and tmp should be writable.